src/App.css

```diff
@@ -46,21 +54,25 @@
 }
 
 .bg-super {
-  background-color: #0000cd !important; /* medium blue #0000CD*/
+  background-color: #0000cd !important;
```

Comment:
Why are we using `!important` here? I'm assuming it's because we are overriding bootstrap default styles. correct me if I am wrong?

Reply:
It isn't my changes:( You're right, there are a lot of linting changes. But I cannot just revert them